### ISSUE TITLE
Support for metadata.json in cli-migrations/v3/docker-entrypoint.sh

### ISSUE THREAD

**Issue by AndresPrez** [hasura/graphql-engine#8423]
So the hasura cli-migrations/v3 script doesnt support loading metadata.json file, only yml files in a metadata directory.

There was an attempt to do this back in the old days https://github.com/hasura/graphql-engine/pull/1968

**Comment by rikinsk** [comment]
Hey, support for this was added in https://github.com/hasura/graphql-engine/commit/186baa7915a59b0ec2cb19de6a0494e60b767fa7. YOu can set a `metadata_file` variable to your `config.yaml` to point to a metadata json file. Unfortunately its not been documented well at the moment. Do let us know if this solves your issue

**Comment by AndresPrez** [comment]
@rikinsk yeah... doesn't look like it work. At least not in `hasura/graphql-engine:v2.5.0.cli-migrations-v3`, I tried that but I still get this error saying that it cant find the metadata:

```
{
    "path": "$.args.metadata",
    "error": "key \\"tables\\" not found",
    "code": "parse-failed"
}
```

**Comment by rikinsk** [comment]
@AndresPrez So it seems like we don't seem to be supporting the metadata file option in the cli-migrations image at the moment. We'll use this issue to track this missing functionality and prioritize it.

**Comment by mbrevda** [comment]
> @AndresPrez So it seems like we don't seem to be supporting the metadata file option in the cli-migrations image at the moment. We'll use this issue to track this missing functionality and prioritize it.

Was tracking metadata as json ever addressed?

**Comment by geoffatsource** [comment]
I'd like to know this as well, as the JSON format is more portable between environments

**Comment by banool** [comment]
I'd love to have this functionality too.

**Comment by h45hc47** [comment]
Want this too, would be nice
